### ISSUE TITLE
(local-exec): ls: cannot open directory '/root': Permission denied

### ISSUE THREAD

**Issue by raviram203** [HSBawa/icp-ce-on-linux-containers#15]
Captured console output of `terraform apply`. Have attached file. I could not diagnose:
```
module.container-master_worker.lxd_container.icp_ce_master (local-exec): ls: cannot open directory '/root': Permission denied
``` 
Suggestions? Profiles & network look okay. Five LXC containers are running with two interfaces 10... & 172... . I am able exec into a container. It has `/root` directory. I could not determine where `ls /root` command originated.

[icp-install-error.txt](https://github.com/HSBawa/icp-ce-on-linux-containers/files/2774976/icp-install-error.txt)

Overhauled my disk allocations & names (complete rebuild of my Ubuntu 18.04 machine). File systems names are aligned with your repo (to minimize tweaks before I build).

**Comment by HSBawa** [comment]
@raviram203  I have commented code causing the issue you encountered (got checked in accidentally). 

Please try the build now.

**Comment by raviram203** [comment]
Thanks, Hari. Worked. Build took 53 minutes. I used `install.sh` for the successful run. Even with the latest code, the alternative route running `terraform apply` failed. Build completed (without an error message) but ICP was not installed. I tried to clean the ICP folder in dev-master-0 but stop/start (as in Section 3) failed to start docker in the containers -- the 172. interface did not come up. Looks like you put some secret sauce  in `install.sh`.
